blockio-api/src-windows/System/FS/BlockIO/Internal.hs

```diff
@@ -28,3 +45,18 @@ hAdvise _h _off _len _advice = pure ()
 
 hAllocate :: Handle HandleIO -> FileOffset -> FileOffset -> IO ()
 hAllocate _h _off _len = pure ()
+
+hSynchronise :: Handle HandleIO -> IO ()
+hSynchronise h = FS.withOpenHandle "hAdvise" (handleRaw h) $ \hANDLE ->
+    Windows.flushFileBuffers hANDLE
```

Comment:
It's a small nit pick but I find the `hANDLE` variable a bit jarring to read. Why not `handle`?

Reply:
No particular reason other than that `Win32` likes all-caps type identifiers :P `hANDLE :: HANDLE`

Reply:
I'll change it